reg/drone/phy/thermodynamics/PressureTempCovTs.0.1.uavcan

```diff
@@ -0,0 +1,15 @@
+# Timestamped fluid pressure and temperature (sampled synchronously) with covariance.
+# Observe that this is a structural subtype of uavcan.si.sample.pressure.Scalar.1.0.
+
+uavcan.time.SynchronizedTimestamp.1.0 timestamp
+
+uavcan.si.unit.pressure.Scalar.1.0    pressure
+uavcan.si.unit.temperature.Scalar.1.0 temperature
+
+float16[3] covariance
+# The upper-right triangle of the covariance matrix (following the matrix packing rules defined in Specification).
```

Comment:
I think you are messing URT with the covariance matrix diagonal (which represents the variances.

| xx | xy | xz |
| ... | yy | yz |
| ... | ... | zz |

An acceptable convention would be using a row-major representation of it. Which results in:
`float16[6] covariance = [xx, xy, xz, yy, yz, zz]`

This usage of the URT for representing the covariances is only acceptable because, as you know, the covariance matrices are symmetric.

Reply:
Same -- you need to notice that the matrix is 2x2, so the URT contains 3 elements.